### ISSUE TITLE
The EOS overlay won't enable, which in return will not allow me to launch a game that requires it.

### ISSUE THREAD

**Issue by FlawlessCasual17** [Heroic-Games-Launcher/HeroicGamesLauncher#3684]
### Describe the bug

The title basically says it all. The EOS overlay (downloaded via lengendary) won't enable, which in turn will not allow me to launch a game that requires it (e.g. The Evil Within).
As a result, I have been forced to resort to the dreaded Epic Games Launcher in order to play any **Epic Store** game.

Here are two of my log files, they will be attached to this report.

**The Evil Within**:

[591e049d72b349daa64784a171379357-lastPlay.log](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/files/14898875/591e049d72b349daa64784a171379357-lastPlay.log)

**Legendary CLI**:

[2024-04-07T20_20_06-legendary.log](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/files/14898881/2024-04-07T20_20_06-legendary.log)

### Add logs

log 1:

```text
Launching "The Evil Within" (legendary)
Native? Yes
Installed in: E:\HeroicGames\TheEvilWithin

System Info:
CPU: 12x 11th Gen Intel(R) Core(TM) i5-11600K @ 3.90GHz
Memory: 17.06 GB (used: 14.9 GB)
GPUs:
  GPU 0:
    Name: NVIDIA Corporation TU116 [GeForce GTX 1660]
    IDs: D=2184 V=10de SD=3fc8 SV=1458
    Driver: 552.12
OS: Microsoft Windows 11 Pro 10.0.22631 (win32)

The current system is not a Steam Deck
We are not running inside a Flatpak container

Software Versions:
  Heroic: 2.14.1 Flame Emperor Sabo
  Legendary: 0.20.34 Direct Intervention
  gogdl: 1.0.1
  Nile: 1.0.2 Jonathan Joestar

Game Settings: {
	"autoInstallDxvk": false,
	"autoInstallDxvkNvapi": false,
	"autoInstallVkd3d": false,
	"preferSystemLibs": false,
	"autoSyncSaves": false,
	"enableEsync": true,
	"enableMsync": false,
	"enableFsync": false,
	"nvidiaPrime": false,
	"enviromentOptions": [],
	"wrapperOptions": [],
	"savesPath": "C:\\Users\\krist\\Saved Games\\TangoGameworks\\The Evil Within (Epic)",
	"showFps": false,
	"useGameMode": false,
	"battlEyeRuntime": false,
	"eacRuntime": false,
	"language": "",
	"beforeLaunchScriptPath": "",
	"afterLaunchScriptPath": ""
}

Game launched at: Sun Apr 07 2024 14:20:45 GMT-0600 (Mountain Daylight Time)

Launch Command: HEROIC_APP_NAME=591e049d72b349daa64784a171379357 HEROIC_APP_RUNNER=legendary HEROIC_APP_SOURCE=epic C:\Users\krist\scoop\apps\legendary\current\legendary.exe launch 591e049d72b349daa64784a171379357 --language en

Game Log:
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching 591e049d72b349daa64784a171379357...
============= End of log =============
```

log 2:
```text

[4/7/2024, 2:20:07 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\heroic-games-launcher\2.14.1\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -NoNewWindow -ArgumentList "`"cleanup`""
[4/7/2024, 2:20:07 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\heroic-games-launcher\2.14.1\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -NoNewWindow -ArgumentList "`"list`"","`"--third-party`""
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Cleanup complete! Removed 0.00 MiB.
[cli] INFO: Getting game list... (this may take a while)

Available games:
 * 20 Minutes Till Dawn (App name: 4656facc740742a39e265b026e13d075 | Version: 1.0-win)
 * A Plague Tale: Innocence (App name: e0fa47ae79514345823bff209ae29451 | Version: EGS_1.08.4)
 * Black Book (App name: 1d74dcbca79540efb52cab2ff026215f | Version: 1.0.51win64)
 * Bloons TD 6 (App name: 7786b355a13b47a6b3915335117cd0b2 | Version: 41.2.7621)
 * Call of the Wild: The Angler™ (App name: fe4168b810314c2fb93f1803e2fe431e | Version: tortoise_delivery_2734622_2734622)
 * Dauntless (App name: Jackal | Version: 1.14.6.615145)
 * Death Stranding (App name: Boga | Version: 2742586.206.1)
 * Death Stranding Content (App name: 5bf67f3153364181b54cb9e40ca7fa82 | Version: 1.01)
  + DEATH STRANDING DIGITAL SOUNDTRACK (App name: 84067e3ec8f149b2bbf2342d7a136065 | Version: 1.02)
  + HD Wallpaper (App name: f8bf1e29bbef480a9b28115ad0d644a0 | Version: 1.02)
  + SELECTIONS OF TITAN ART BOOK (App name: c311b22f878749449886949eeb588999 | Version: 1.01)
 * Deliver Us Mars (App name: 9d943a160fa841859ac5c1f233789fcb | Version: 2.0035387)
 * Destiny 2 (App name: 592c359fb0e0413fb46dee2d24448eb4 | Version: 9189690)
 * Deus Ex - Mankind Divided (App name: 2eaba703e63e4ed8bd2919ed552021d6 | Version: DXMD_EPIC_1.05)
 * Dishonored - Definitive Edition (App name: d2da64bd4c4e422da4b1a88041492a3a | Version: 1.6)
 * Dishonored®: Death of the Outsider™ (App name: 2fb8273dcf6f41e4899c0c881e047053 | Version: 0.0.0.4)
 * Doors - Paradox (App name: 91e82f342ebf42309703e9ba363daedf | Version: 1.00)
 * Dying Light Enhanced Edition (App name: 04a54ed532ce4fbfbf8fd5833f60def0 | Version: DL_1.49.8f4)
  + Dying Light The Bozak (App name: 71816b992ef04030b1c58af6b9845fe5 | Version: BOZ_1.49.8f4)
  + Dying Light The Following (App name: f653266870894ee1acbb5250e3b04bd1 | Version: FOLL_1.49.8f4)
 * Europa Universalis IV (App name: da0103e959e54d139d0c109ded3b3672 | Version: EU4-v1.36.2.0-Byzantium-e93d-windows-live-248)
 * Fallout 3: Game of the Year Edition (App name: adeae8bbfc94427db57c7dfecce3f1d4 | Version: 1.11)
 * Fortnite (App name: Fortnite | Version: ++Fortnite+Release-29.10-CL-32567225-Windows)
 * Genshin Impact (App name: 41869934302e4b8cafac2d3c0e7c293d | Version: 87)
 * GRIME (App name: 028263f54a874e35800d142275254d2c | Version: 1.3.5)
 * Islets (App name: 5e37739a766b4457981d2992f93ac72c | Version: 1.03)
 * Loop Hero (App name: f7a0ebb44f93430fb1c4388a395eba96 | Version: 11054)
 * Marvel's Guardians of the Galaxy (App name: 63a665088eb1480298f1e57943b225d8 | Version: 2984448)
  + Marvel's Guardians of the Galaxy: Social-Lord Outfit (App name: 9596620d263b40c083ddf1f0c662c8ff | Version: 2978210)
 * Never Alone (Kisima Ingitchuna) (App name: f38de9fa4c6d43eabac080942cb394a1 | Version: NeverAlone-Win-1.2.8)
 * Sail Forth (App name: 11ba63f98ce94a38b631104a12c7adab | Version: Win_2024-02-07_13-13-35)
 * Saints Row (App name: Cormorant | Version: Manor_Wrapper_Update_RC1_CL4735700-230829_0259UTC)
 * Severed Steel (App name: 7179f095efcc4f92a950a6d9dbd9c602 | Version: 5.5.0)
 * Shadow of the Tomb Raider: Definitive Edition (App name: 890d9cf396d04922a1559333df419fed | Version: TR11_10072023_492.0_1227914_740629_133334955480774690_)
 * Super Meat Boy Forever (App name: Moose | Version: 6755.1848.1962.152)
 * The Dungeon Of Naheulbeuk: The Amulet Of Chaos (App name: 02fac38ee2614c8ba276b1ea7c1acd7c | Version: Windows_1.5_575_47857)
 * The Elder Scrolls Online (App name: 4d0ff75b922447649057c237c0bd1545 | Version: EGSStarter_v2.9_d6_live_public)
 * The Evil Within (App name: 591e049d72b349daa64784a171379357 | Version: 0.0.0.6)
 * The Evil Within 2 (App name: 74977fee80de4e5e9f663e62560a0f2f | Version: 0.0.0.5)
 * The Outer Worlds: Spacer's Choice Edition (App name: cb3bf7ba89574a66ae3b795e039d4dbc | Version: 0.0.0.6415)
 * The Silent Age (App name: 19a750a7d7974f9291c0584feedb94b0 | Version: 1.0-win)
 * Thief (App name: 4123bcce398b4b1d8b9ef878f81055d1 | Version: TH4_EPIC_1.7.0.3)
 * Tunche (App name: fd51551d919847beb178985f6daf0306 | Version: v1.1.0.0)
 * Turnip Boy Commits Tax Evasion (App name: 3d34f303a68c44c2a9b44a3988533e90 | Version: 1.1.2f2)

Total: 38
[4/7/2024, 2:20:13 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\heroic-games-launcher\2.14.1\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -NoNewWindow -ArgumentList "`"list`"","`"--third-party`""
[4/7/2024, 2:20:14 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\heroic-games-launcher\2.14.1\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -NoNewWindow -ArgumentList "`"--version`""
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
legendary version "0.20.34", codename "Direct Intervention"
[cli] INFO: Getting game list... (this may take a while)

Available games:
 * 20 Minutes Till Dawn (App name: 4656facc740742a39e265b026e13d075 | Version: 1.0-win)
 * A Plague Tale: Innocence (App name: e0fa47ae79514345823bff209ae29451 | Version: EGS_1.08.4)
 * Black Book (App name: 1d74dcbca79540efb52cab2ff026215f | Version: 1.0.51win64)
 * Bloons TD 6 (App name: 7786b355a13b47a6b3915335117cd0b2 | Version: 41.2.7621)
 * Call of the Wild: The Angler™ (App name: fe4168b810314c2fb93f1803e2fe431e | Version: tortoise_delivery_2734622_2734622)
 * Dauntless (App name: Jackal | Version: 1.14.6.615145)
 * Death Stranding (App name: Boga | Version: 2742586.206.1)
 * Death Stranding Content (App name: 5bf67f3153364181b54cb9e40ca7fa82 | Version: 1.01)
  + DEATH STRANDING DIGITAL SOUNDTRACK (App name: 84067e3ec8f149b2bbf2342d7a136065 | Version: 1.02)
  + HD Wallpaper (App name: f8bf1e29bbef480a9b28115ad0d644a0 | Version: 1.02)
  + SELECTIONS OF TITAN ART BOOK (App name: c311b22f878749449886949eeb588999 | Version: 1.01)
 * Deliver Us Mars (App name: 9d943a160fa841859ac5c1f233789fcb | Version: 2.0035387)
 * Destiny 2 (App name: 592c359fb0e0413fb46dee2d24448eb4 | Version: 9189690)
 * Deus Ex - Mankind Divided (App name: 2eaba703e63e4ed8bd2919ed552021d6 | Version: DXMD_EPIC_1.05)
 * Dishonored - Definitive Edition (App name: d2da64bd4c4e422da4b1a88041492a3a | Version: 1.6)
 * Dishonored®: Death of the Outsider™ (App name: 2fb8273dcf6f41e4899c0c881e047053 | Version: 0.0.0.4)
 * Doors - Paradox (App name: 91e82f342ebf42309703e9ba363daedf | Version: 1.00)
 * Dying Light Enhanced Edition (App name: 04a54ed532ce4fbfbf8fd5833f60def0 | Version: DL_1.49.8f4)
  + Dying Light The Bozak (App name: 71816b992ef04030b1c58af6b9845fe5 | Version: BOZ_1.49.8f4)
  + Dying Light The Following (App name: f653266870894ee1acbb5250e3b04bd1 | Version: FOLL_1.49.8f4)
 * Europa Universalis IV (App name: da0103e959e54d139d0c109ded3b3672 | Version: EU4-v1.36.2.0-Byzantium-e93d-windows-live-248)
 * Fallout 3: Game of the Year Edition (App name: adeae8bbfc94427db57c7dfecce3f1d4 | Version: 1.11)
 * Fortnite (App name: Fortnite | Version: ++Fortnite+Release-29.10-CL-32567225-Windows)
 * Genshin Impact (App name: 41869934302e4b8cafac2d3c0e7c293d | Version: 87)
 * GRIME (App name: 028263f54a874e35800d142275254d2c | Version: 1.3.5)
 * Islets (App name: 5e37739a766b4457981d2992f93ac72c | Version: 1.03)
 * Loop Hero (App name: f7a0ebb44f93430fb1c4388a395eba96 | Version: 11054)
 * Marvel's Guardians of the Galaxy (App name: 63a665088eb1480298f1e57943b225d8 | Version: 2984448)
  + Marvel's Guardians of the Galaxy: Social-Lord Outfit (App name: 9596620d263b40c083ddf1f0c662c8ff | Version: 2978210)
 * Never Alone (Kisima Ingitchuna) (App name: f38de9fa4c6d43eabac080942cb394a1 | Version: NeverAlone-Win-1.2.8)
 * Sail Forth (App name: 11ba63f98ce94a38b631104a12c7adab | Version: Win_2024-02-07_13-13-35)
 * Saints Row (App name: Cormorant | Version: Manor_Wrapper_Update_RC1_CL4735700-230829_0259UTC)
 * Severed Steel (App name: 7179f095efcc4f92a950a6d9dbd9c602 | Version: 5.5.0)
 * Shadow of the Tomb Raider: Definitive Edition (App name: 890d9cf396d04922a1559333df419fed | Version: TR11_10072023_492.0_1227914_740629_133334955480774690_)
 * Super Meat Boy Forever (App name: Moose | Version: 6755.1848.1962.152)
 * The Dungeon Of Naheulbeuk: The Amulet Of Chaos (App name: 02fac38ee2614c8ba276b1ea7c1acd7c | Version: Windows_1.5_575_47857)
 * The Elder Scrolls Online (App name: 4d0ff75b922447649057c237c0bd1545 | Version: EGSStarter_v2.9_d6_live_public)
 * The Evil Within (App name: 591e049d72b349daa64784a171379357 | Version: 0.0.0.6)
 * The Evil Within 2 (App name: 74977fee80de4e5e9f663e62560a0f2f | Version: 0.0.0.5)
 * The Outer Worlds: Spacer's Choice Edition (App name: cb3bf7ba89574a66ae3b795e039d4dbc | Version: 0.0.0.6415)
 * The Silent Age (App name: 19a750a7d7974f9291c0584feedb94b0 | Version: 1.0-win)
 * Thief (App name: 4123bcce398b4b1d8b9ef878f81055d1 | Version: TH4_EPIC_1.7.0.3)
 * Tunche (App name: fd51551d919847beb178985f6daf0306 | Version: v1.1.0.0)
 * Turnip Boy Commits Tax Evasion (App name: 3d34f303a68c44c2a9b44a3988533e90 | Version: 1.1.2f2)

Total: 38
[4/7/2024, 2:20:22 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\heroic-games-launcher\2.14.1\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -NoNewWindow -ArgumentList "`"--version`""
[4/7/2024, 2:20:24 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\heroic-games-launcher\2.14.1\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -NoNewWindow -ArgumentList "`"--version`""
[4/7/2024, 2:20:24 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\heroic-games-launcher\2.14.1\resources\app.asar.unpacked\build\bin\win32\legendary`"" -Wait -NoNewWindow -ArgumentList "`"eos-overlay`"","`"info`""
legendary version "0.20.34", codename "Direct Intervention"
[cli] INFO: Installed version: 1.2.7++EOSSDK+Release-Overlay-CL-25975892-Windows
[cli] INFO: Installed path: C:\Users\krist\AppData\Roaming\heroic\tools\eos_overlay
[cli] INFO: Found available Overlay installations in:
[cli] INFO:  - C:\Users\krist\AppData\Roaming\heroic\tools\eos_overlay
[cli] INFO: Overlay enabled: Yes
[cli] INFO: Enabled Overlay path: C:/Users/krist/AppData/Roaming/heroic/tools/eos_overlay
[cli] INFO: Enabled Vulkan layers:
[cli] INFO:  - C:\Users\krist\AppData\Roaming\heroic\tools\eos_overlay\EOSOverlayVkLayer-Win32.json
[cli] INFO:  - C:\Users\krist\AppData\Roaming\heroic\tools\eos_overlay\EOSOverlayVkLayer-Win64.json
[4/7/2024, 2:20:27 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\legendary\current\legendary.exe`"" -Wait -NoNewWindow -ArgumentList "`"--version`""
legendary version "0.20.34", codename "Direct Intervention"
[4/7/2024, 2:20:31 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\legendary\current\legendary.exe`"" -Wait -NoNewWindow -ArgumentList "`"status`""
[Core] INFO: Trying to re-use existing login session...
Epic account: Zliced13
Games available: 38
Games installed: 1
EGL Sync enabled: True
Config directory: C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary
Platform (System): Windows-10-10.0.22631-SP0 (nt)

Legendary version: 0.20.34 - "Direct Intervention"
Update available: no
[4/7/2024, 2:20:40 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\legendary\current\legendary.exe`"" -Wait -NoNewWindow -ArgumentList "`"eos-overlay`"","`"remove`"","`"-y`""
[cli] INFO: Deleting overlay installation...
[cli] INFO: Removing registry entries...
[cli] INFO: Done.
[4/7/2024, 2:20:45 PM] HEROIC_APP_NAME=591e049d72b349daa64784a171379357 HEROIC_APP_RUNNER=legendary HEROIC_APP_SOURCE=epic LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\legendary\current\legendary.exe`"" -Wait -NoNewWindow -ArgumentList "`"launch`"","`"591e049d72b349daa64784a171379357`"","`"--language`"","`"en`""
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching 591e049d72b349daa64784a171379357...
[4/7/2024, 2:20:50 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\legendary\current\legendary.exe`"" -Wait -NoNewWindow -ArgumentList "`"list`"","`"--third-party`""
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Getting game list... (this may take a while)

Available games:
 * 20 Minutes Till Dawn (App name: 4656facc740742a39e265b026e13d075 | Version: 1.0-win)
 * A Plague Tale: Innocence (App name: e0fa47ae79514345823bff209ae29451 | Version: EGS_1.08.4)
 * Black Book (App name: 1d74dcbca79540efb52cab2ff026215f | Version: 1.0.51win64)
 * Bloons TD 6 (App name: 7786b355a13b47a6b3915335117cd0b2 | Version: 41.2.7621)
 * Call of the Wild: The Angler™ (App name: fe4168b810314c2fb93f1803e2fe431e | Version: tortoise_delivery_2734622_2734622)
 * Dauntless (App name: Jackal | Version: 1.14.6.615145)
 * Death Stranding (App name: Boga | Version: 2742586.206.1)
 * Death Stranding Content (App name: 5bf67f3153364181b54cb9e40ca7fa82 | Version: 1.01)
  + DEATH STRANDING DIGITAL SOUNDTRACK (App name: 84067e3ec8f149b2bbf2342d7a136065 | Version: 1.02)
  + HD Wallpaper (App name: f8bf1e29bbef480a9b28115ad0d644a0 | Version: 1.02)
  + SELECTIONS OF TITAN ART BOOK (App name: c311b22f878749449886949eeb588999 | Version: 1.01)
 * Deliver Us Mars (App name: 9d943a160fa841859ac5c1f233789fcb | Version: 2.0035387)
 * Destiny 2 (App name: 592c359fb0e0413fb46dee2d24448eb4 | Version: 9189690)
 * Deus Ex - Mankind Divided (App name: 2eaba703e63e4ed8bd2919ed552021d6 | Version: DXMD_EPIC_1.05)
 * Dishonored - Definitive Edition (App name: d2da64bd4c4e422da4b1a88041492a3a | Version: 1.6)
 * Dishonored®: Death of the Outsider™ (App name: 2fb8273dcf6f41e4899c0c881e047053 | Version: 0.0.0.4)
 * Doors - Paradox (App name: 91e82f342ebf42309703e9ba363daedf | Version: 1.00)
 * Dying Light Enhanced Edition (App name: 04a54ed532ce4fbfbf8fd5833f60def0 | Version: DL_1.49.8f4)
  + Dying Light The Bozak (App name: 71816b992ef04030b1c58af6b9845fe5 | Version: BOZ_1.49.8f4)
  + Dying Light The Following (App name: f653266870894ee1acbb5250e3b04bd1 | Version: FOLL_1.49.8f4)
 * Europa Universalis IV (App name: da0103e959e54d139d0c109ded3b3672 | Version: EU4-v1.36.2.0-Byzantium-e93d-windows-live-248)
 * Fallout 3: Game of the Year Edition (App name: adeae8bbfc94427db57c7dfecce3f1d4 | Version: 1.11)
 * Fortnite (App name: Fortnite | Version: ++Fortnite+Release-29.10-CL-32567225-Windows)
 * Genshin Impact (App name: 41869934302e4b8cafac2d3c0e7c293d | Version: 87)
 * GRIME (App name: 028263f54a874e35800d142275254d2c | Version: 1.3.5)
 * Islets (App name: 5e37739a766b4457981d2992f93ac72c | Version: 1.03)
 * Loop Hero (App name: f7a0ebb44f93430fb1c4388a395eba96 | Version: 11054)
 * Marvel's Guardians of the Galaxy (App name: 63a665088eb1480298f1e57943b225d8 | Version: 2984448)
  + Marvel's Guardians of the Galaxy: Social-Lord Outfit (App name: 9596620d263b40c083ddf1f0c662c8ff | Version: 2978210)
 * Never Alone (Kisima Ingitchuna) (App name: f38de9fa4c6d43eabac080942cb394a1 | Version: NeverAlone-Win-1.2.8)
 * Sail Forth (App name: 11ba63f98ce94a38b631104a12c7adab | Version: Win_2024-02-07_13-13-35)
 * Saints Row (App name: Cormorant | Version: Manor_Wrapper_Update_RC1_CL4735700-230829_0259UTC)
 * Severed Steel (App name: 7179f095efcc4f92a950a6d9dbd9c602 | Version: 5.5.0)
 * Shadow of the Tomb Raider: Definitive Edition (App name: 890d9cf396d04922a1559333df419fed | Version: TR11_10072023_492.0_1227914_740629_133334955480774690_)
 * Super Meat Boy Forever (App name: Moose | Version: 6755.1848.1962.152)
 * The Dungeon Of Naheulbeuk: The Amulet Of Chaos (App name: 02fac38ee2614c8ba276b1ea7c1acd7c | Version: Windows_1.5_575_47857)
 * The Elder Scrolls Online (App name: 4d0ff75b922447649057c237c0bd1545 | Version: EGSStarter_v2.9_d6_live_public)
 * The Evil Within (App name: 591e049d72b349daa64784a171379357 | Version: 0.0.0.6)
 * The Evil Within 2 (App name: 74977fee80de4e5e9f663e62560a0f2f | Version: 0.0.0.5)
 * The Outer Worlds: Spacer's Choice Edition (App name: cb3bf7ba89574a66ae3b795e039d4dbc | Version: 0.0.0.6415)
 * The Silent Age (App name: 19a750a7d7974f9291c0584feedb94b0 | Version: 1.0-win)
 * Thief (App name: 4123bcce398b4b1d8b9ef878f81055d1 | Version: TH4_EPIC_1.7.0.3)
 * Tunche (App name: fd51551d919847beb178985f6daf0306 | Version: v1.1.0.0)
 * Turnip Boy Commits Tax Evasion (App name: 3d34f303a68c44c2a9b44a3988533e90 | Version: 1.1.2f2)

Total: 38
[4/7/2024, 2:20:54 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\legendary\current\legendary.exe`"" -Wait -NoNewWindow -ArgumentList "`"--version`""
[4/7/2024, 2:20:54 PM] LEGENDARY_CONFIG_PATH=C:\Users\krist\AppData\Roaming\heroic\legendaryConfig\legendary powershell Start-Process "`"C:\Users\krist\scoop\apps\legendary\current\legendary.exe`"" -Wait -NoNewWindow -ArgumentList "`"eos-overlay`"","`"info`""
legendary version "0.20.34", codename "Direct Intervention"
[cli] INFO: No Legendary-managed installation found.
[cli] INFO: Found available Overlay installations in:
[cli] INFO: Overlay enabled: No
[cli] INFO: Enabled Overlay path: None
[cli] INFO: No enabled Vulkan layers.
```

### Steps to reproduce

1. Go to Open the HGL.
2. Click on any game that is downloaded from the **Epic Store**
3. Wait for the game to launch.

### Expected behavior

I was expecting “The Evil Within” and **Epic Store** games like it to launch without ANY issues.

### Screenshots

![(Heroic)F7h7H4_04-07-2024](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/101912712/5be0cb4e-4a89-4ae9-a789-8533298d2e15)

### Heroic Version

Latest Stable

### System Information

- OS [e. g. "Windows 11 Pro"]:

### Additional information

I use a FOSS firewall app called Safing Portmaster to monitor my network connections, and block unwanted domains.

Sometimes the EOS overlay turns off and then turns back off, I don't know why it is doing that.

**Comment by FlawlessCasual17** [comment]
I did find a really simple workaround for this problem (It's not a solution, and it only works for users of Windows 10/11). If you install the ***Epic Games Launcher***. An **Epic Store** game like “The Evil Within” will use the overlay from the Epic Games Launcher instead of using the one downloaded via `lengendary`.

**Comment by FlawlessCasual17** [comment]
I solved the problem, turns you just need to use this Windows registry file from this website, https://github.com/derrod/legendary/wiki/Dishonored-Death-of-the-Outsider

But before I did that, I stopped the heroic games launcher first. Then I used the Windows registry file from the specified website.

Afterwards, I opened the heroic games launcher and clicked the “The Evil Within”. The game itself launched perfectly.
